docs: Migrate Existing Tasks page for ECS (#11048)
hashicorp/consul: website/content/docs/ecs/get-started/install.mdx

    }
  ]

  port       = "9090"
  retry_join = "<address of the Consul server>"
}
```

All possible inputs are documented on the [module reference documentation](https://registry.terraform.io/modules/hashicorp/consul-ecs/aws/latest/submodules/mesh-task?tab=inputs),
however there are some important inputs worth highlighting:

- `family` is used as the [task definition family](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#family)
  but it's also used as the name of the service that gets registered in Consul.
- `container_definitions` accepts an array of [container definitions](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definitions).
  This is where you include application containers.
- `port` is the port that your application listens on. This should be set to a
  string, not an integer, i.e. `port = "9090"`, not `port = 9090`.

> - `retry_join` is passed to the `-retry-join` option for the Consul agent. This tells

Reviewer: ```suggestion
- `retry_join` is passed to the [`-retry-join`](/docs/agent/options#_retry_join) option for the Consul agent. This tells
```

needs to call. You then must modify your application to make requests to the sidecar
proxy on that port.

> For example, say your application `web` wants to make calls to my other application

Reviewer 2: ```suggestion
For example, if your application `web` makes calls to another application called `backend`, then you would first configure the `mesh-task` module's upstream(s):
```
GitHub wouldn't let me make a suggestion that spans these lines.

`backend`.

> First, you must configure the `mesh-task` module's upstream(s):

Reviewer 2: ```suggestion

```
Merged into the previous comment.
